ArduPlane/Parameters.cpp

```diff
@@ -1058,10 +1058,26 @@ const AP_Param::Info Plane::var_info[] = {
     GOBJECT(relay,                  "RELAY_", AP_Relay),
 
 #if PARACHUTE == ENABLED
-	// @Group: CHUTE_
+    // @Group: CHUTE_
     // @Path: ../libraries/AP_Parachute/AP_Parachute.cpp
     GOBJECT(parachute,		"CHUTE_", AP_Parachute),
 
+    // @Param: CHUTE_AUTO_ON
+    // @DisplayName: Parachute automatic emergency release
+    // @Description: Parachute automatic emergency release enabled or disabled.
+    // @Values: 0:Disabled,1:Enabled
+    // @User: Standard
+    GSCALAR(parachute_auto_enabled,    "CHUTE_AUTO_ON",   0),
+
+    // @Param: CHUTE_AUTO_ERROR
+    // @DisplayName: Altitude deviation for parachute release
+    // @Description: Altitude deviation at which to release parachute if in AUTO and CHUTE_AUTO_ON.
+    // @Units: Meters
+    // @Range: 10 32767
+    // @Increment: 1
+    // @User: Standard
+    GSCALAR(parachute_auto_error,      "CHUTE_AUTO_ERROR",      20),
```

Comment:
When you move these to AP_Parachute, you can surround them with # if APM_BUILD_TYPE(APM_BUILD_ArduPlane) so they only compile for plane. Same goes for the timer variable and accessor function.